### PR TITLE
Update UG and DG

### PR DESCRIPTION
**Pull request by WM71811**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -682,7 +682,7 @@ with phone number, using a prefix to identify them.
 * needs to keep track of the daily sales
 * prefers desktop apps over other types
 * prefers typing to mouse interactions
-* is reasonably comfortable using CLI apps
+* is reasonably comfortable using command-line interface (CLI) apps
```
Comment (ureshiiYing) [inline docs/DeveloperGuide.md]: No need to write the long form here, I have added the long form in the introduction

File: docs/UserGuide.md [hunk 2]
```diff
@@ -43,23 +43,13 @@ You can start from the [Quick Start](#quick-start) section to learn how to obtai
 
 1. Copy the file to the folder you want to use as the _home folder_ for your tCheck.
 
-1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note
+1. Double-click the file to start the app. The graphic user interface (GUI) similar to the figure below should appear in a few seconds. Note
```
Comment (ureshiiYing) [inline docs/UserGuide.md:46]: Should be Graphical User Interface (capitalise the first letter)

**Review by yanbingtao — DISMISSED** [review]
LGTM

File: docs/UserGuide.md (hunk 2 above)
Comment (WM71811) [reply docs/UserGuide.md:46]: Thanks, I think not capitalising would be also accepted.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (WM71811) [reply docs/DeveloperGuide.md]: Okay thanks

**Review by yanbingtao — APPROVED** [review]
LGTM

**Review by ureshiiYing — APPROVED** [review]
LGTM

**Pull request merged**